src/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const artistRouter = require('./routes/artist')
+
+const app = express();
+
+app.use(express.json());
+
+app.use('/artists', artistRouter);
+
+
+module.exports = app;
```

Comment:
It would be fine to condense this a bit and group the various bits of functionality together:
```
const express = require('express');
const artistRouter = require('./routes/artist');

const app = express();
app.use(express.json());
app.use('/artists', artistRouter);

module.exports = app;
```